rpm.pc.in

```diff
@@ -8,7 +8,7 @@ Name: RPM
 Description: RPM Package Manager
 Version: @VERSION@
 URL: http://rpm.org
-# Requires: 
+Requires.private: @ZSTD_REQUIRES@
```

Comment:
this is mostly if you would like to do `pkg-config --static --cflags --ldflags rpm` to link RPM statically.. most likely it will not work at all due to some other issues, but this is not problem of this patchset ;)